file/nifile.yml

```diff
@@ -0,0 +1,816 @@
+      parameters:
+        - in: query
+          name: skip
+          description: How many files to skip in the result when paging
+          type: integer
+          x-ni-data-type: U32
+          default: 0
+        - in: query
+          name: take
+          description: >-
+            How many files to return in the result, or 0 to use a default
+            defined by the service
+          type: integer
+          x-ni-data-type: U32
+          default: 0
+          maximum: 10000
+        - in: query
+          name: orderBy
+          description: The name of the metadata key to sort by
+          type: string
+          enum:
+            - created
+            - id
+            - size
+        - in: query
+          name: orderByDescending
+          description: Whether to sort descending instead of ascending
+          type: boolean
+          default: false
+        - in: query
+          name: id
+          description: Comma-separated list of file IDs to restrict to in the results
+          type: string
+          x-example: '5afb2ce3741fe11d88838cc9,4afb2ce3741fe11d88838cc9'
+      responses:
+        200:
+          $ref: '#/responses/Query'
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/service-groups/Default/files/{id}:
+    parameters:
+      - $ref: '#/parameters/id'
+    delete:
+      tags: [files]
+      summary: Delete file
+      description: Deletes the file indicated by the resource ID.
+      operationId: Delete
+      x-ni-operation: deleteFiles
+      x-ni-privilege: ModifyFiles
+      responses:
+        204:
+          description: No content
+        401:
+          $ref: '#/responses/Unauthorized'
+        default:
+          $ref: '#/responses/Error'
+  /v1/service-groups/Default/files/{id}/data:
+    parameters:
+      - $ref: '#/parameters/id'
+    get:
+      tags: [files]
+      summary: Download file
+      description: >-
+        Downloads a file from the Skyline File service. The file is returned in
+        a single HTTP response. The file's name metadata property is appended
+        to the response header so that the file is downloaded with the
+        appropriate file name.When the the inline query parameter is not
+        specified or is set to false, the Content-Disposition header is set to
+        'attachment' and the MIME type is set to binary so that the browser
+        handles the response as a file download.  When the inline query
+        parameter is set to true, the Content-Disposition header and MIME type
+        remain unset, and the file contents must be handled by the caller.
+      operationId: ReceiveFile
+      x-ni-operation: downloadData
+      x-ni-privilege: DownloadFiles
+      x-ni-request-variables:
+        - REQUEST_METHOD
+      parameters:
+        - in: query
+          name: inline
+          description: Whether to return the file data inline rather than as an attachment.
```

Comment:
When this value is true, the file contents return inline for the caller to handle. When this value is not specified or false, the browser handles the response as a file download.  

Note: I removed parts about the Content-Disposition header and MIME type, because it doesn't seem like these affect end users much. But I could be wrong. Feel free to add it back in.